src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -755,6 +761,8 @@ public void SaveWallet()
         /// <inheritdoc />
         public bool RemoveTransientTransactions(uint256 depositId = null)
         {
+            this.logger.LogTrace("Removing transient transactions. DepositId={0}", depositId);
```

Comment:
That's odd, Fody should be inserting entry logs here?

Reply:
There were sooo many calls when we had Fody in, we turned it off...

Reply:
Was making like 3.5GB data / node / hour

Reply:
OK so you can just add [NoTrace] to the methods you dont want traced?